packages/testcases/View Guided Answers Content.md

```diff
@@ -78,3 +90,14 @@ Note: Check the content matches the content in the web application e.g. headers
 **THEN** the user is navigated to the Guided Answers home page
     
     a. an empty search 
+    
+ 
+### Displaying enhancements specific to VSCode or SAP Business Application Studio
+
+**GIVEN** the user has opened Guided Answers, searched and selected Fiori Tools in VSCODE
+**WHEN** the user clicks on 'Fior Generator' **AND** 'In SAP Business Application Studio - Services unavailable, All catalog service requests failed'
```

Comment:
Small correction : `Fiori`

Reply:
@theStiggySpd - This is now outdated. Do you know how we correct the spelling mistake or do I have to create a new pull request?